### PR TITLE
UI update (batch of simple components) 

### PR DESCRIPTION
**Pull request by jeremie-lucca**
## Description

Update for basic components

- Chip
- Tag
- Tooltip
- Menu
- Page Header
- Timepicker
- Pagination
- Popover
- Filter

-----
image
image
image
image
image
image
image
image
image
image
-----

### CODE REVIEW

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-2138/storybook

File: packages/scss/src/components/chip/states.scss [hunk 1]
```diff
@@ -0,0 +1,8 @@
+@mixin disabled {
+  background: var(--palettes-grey-100);
+  color: var(--palettes-grey-500);
+
+  .chip-kill {
```
Comment (vvalentin-lucca) [inline packages/scss/src/components/chip/states.scss:5]: Je le laisserais visible et désactivé.

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-2138/storybook

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-2138/storybook

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-2138/storybook

File: packages/scss/src/components/menu/component.scss [hunk 2]
```diff
@@ -32,6 +32,7 @@
 		.menu-list {
 			@include reset.list;
 
+			align-items: end;
```
Comment (vvalentin-lucca) [inline packages/scss/src/components/menu/component.scss:35]: flex-end

**Review by vvalentin-lucca — APPROVED** [review]

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-2138/storybook

**Pull request merged**